docs/data/data-grid/filtering-recipes/filtering-recipes.md

```diff
@@ -13,9 +13,3 @@ Currently if you want to use the [Quick filter](/x/react-data-grid/filtering/qui
 A common use case is to have certain components positioned outside of the grid. Because of the way the grid context works this might not be a straightforward thing to do. The example below illustrates how this use case can be achieved.
```

Comment:
I've added whitespace to force the height of the empty option to avoid the need for l10n:

<img width="625" src="https://github.com/mui/mui-x/assets/13808724/3315d0af-5446-4177-a326-ca19504ae908">